Makefile

```diff
@@ -0,0 +1,10 @@
+.PHONY: build
+build:
+	CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -v -o ./dist/nats-source main.go
+
+.PHONY: image
+image: build
+	docker build -t "quay.io/numaio/numaflow-source/nats-source:v0.5.2" --target nats-source .
+
```

Comment:
should the image name be "nats-source-go" now, to reflect the new name of the repo?